hera_cal/tests/test_utils.py

```diff
@@ -69,4 +71,25 @@ def test_LST2JD(self):
         nt.assert_almost_equal(utils.LST2JD(12.0, 2458042, 21.), 2458042.8720297855)
 
 
+class Test_combine_calfits:
```

Comment:
I'd like to see some more verification tests just asserting that the file exists and the number of antennas is right. I'd like to see that the number of times is correct and that the gain from a particular channel and integration from test_file1 and test_file2 actually ends up in the right place.

Reply:
done